generators/server/templates/src/main/resources/logback-spring.xml.ejs

```diff
@@ -45,6 +45,7 @@
     </root>
 -->
 
+    <logger name="<%= packageName %>" level="INFO"/>
```

Comment:
This is not needed, already defined in application.yml

Reply:
```suggestion
```